src/extension/background-script/actions/permissions/__tests__/add.test.ts

```diff
@@ -1,10 +1,20 @@
 import db from "~/extension/background-script/db";
+import state from "~/extension/background-script/state";
 import { allowanceFixture } from "~/fixtures/allowances";
 import { permissionsFixture } from "~/fixtures/permissions";
 import type { DbAllowance, MessagePermissionAdd } from "~/types";
 
 import addPermission from "../add";
 
+jest.mock("~/extension/background-script/state");
+
+const defaultMockState = {
+  currentAccountId: "123456",
+};
+
+const mockState = defaultMockState;
+state.getState = jest.fn().mockReturnValue(mockState);
+
```

Comment:
add another check that the currentAccountId is stored to the permission.

Reply:
Added a check by changing the permissions fixture: https://github.com/getAlby/lightning-browser-extension/pull/2002/files#diff-851f81c1aba04e58f78a0c6f5e34359aae948bc3cd375f641c0f2b65b811fe94